assets/js/common/ActivityLogsConfig/ActivityLogsConfig.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { noop } from 'lodash';
+
+import Button from '@common/Button';
+
+/**
+ * Simplistic utility to transform a time interval in a readable string
+ */
+function TimeLabel({ time }) {
+  if (typeof time === 'string') {
+    return time;
+  }
+  if (
+    typeof time === 'object' &&
+    time !== null &&
+    'value' in time &&
+    'unit' in time
+  ) {
+    return `${time.value} ${time.unit}${time.value > 1 ? 's' : ''}`;
+  }
+  return <em>invalid value</em>;
+}
+
+/**
+ * Render the saved configuration for the Activity Logs.
+ * Editing is demanded to another component, activated by the Edit button
+ *
+ * @param {{value: number, unit: "days"|"months"|"years"}} props.retentionTime A structured `{value,unit}`
+ * object that defines a retention time interval. `unit` is one of `days`, `months`, `years`.'
+ * @param {function} props.onEditClick Handles the edit button click
+ */
+function ActivityLogConfig({ retentionTime, onEditClick = noop }) {
+  return (
+    <div className="container max-w-7xl mx-auto p-4 sm:p-6 lg:p-8 bg-white dark:bg-gray-800 rounded-lg">
+      <div>
+        <h2 className="text-2xl font-bold inline-block">Activity Logs</h2>
+        <span className="float-right">
+          <Button
+            className="mr-2"
+            type="primary-white-fit"
+            size="small"
+            onClick={onEditClick}
+          >
+            Edit Settings
+          </Button>
+        </span>
+      </div>
+      <p className="mt-3 mb-3 text-gray-500">
+        Activity Log dolor sit amet, consectetur adipiscing elit. Nunc sit amet
+        ex sem. Duis faucibus tellus ipsum, sit amet euismod magna porttitor et
```

Comment:
@jagabomb @abravosuse this is just a placeholder for text, let's defining a proper one if we need it